go-controller/pkg/ovn/egressip.go

```diff
@@ -398,9 +398,15 @@ func (oc *Controller) reconcileEgressIPPod(old, new *v1.Pod) (err error) {
 	}
 
 	// Iterate all EgressIPs and check if this pod start/stops matching any and
-	// add/remove the setup accordingly. Pods cannot match multiple EgressIP
-	// objects: that is considered a user error and is undefined. Hence, just
-	// just perform the setup against the first object matched.
+	// add/remove the setup accordingly. Pods should not match multiple EgressIP
```

Comment:
this commit needs a commit message with a good explanation, probably something summarizing this comment

Reply:
OK, will add that